### DISCUSSION TITLE
Vanjaro Platform Performance

### DISCUSSION THREAD

**Discussion by renderfarmer** [vanjarosoftware/Vanjaro.Platform#2237] [Q&A] [answered]
Does anyone know if there is any performance difference in Site Speed when comparing a self installed website created with Vanjaro Platform instead of Vanjaro for DNN? I mean if you basically create the same page, would there be any difference?

**Comment by meetmandeep** [comment, marked as answer]
You would see better performance with Vanjaro Platform in comparison with Vanjaro for DNN. Vanjaro Platform was specifically created as a minimal install of integrated DNN to boost performance. 

There would be difference in identical pages; Vanjaro Platform would be leaner in terms of preloaded JS/CSS.

**Reply by renderfarmer** [reply]
Ok great. Thanks!
